src/app/modules/auth/__tests__/auth.routes.spec.ts

```diff
@@ -417,7 +417,7 @@ describe('auth.routes', () => {
       expect(results).toEqual(
         Array(AuthService.MAX_OTP_ATTEMPTS).fill({
           status: 422,
-          text: 'OTP is invalid. Please try again.',
+          text: JSON.stringify('OTP is invalid. Please try again.'),
```

Comment:
nit: this can be `body` also, haha

Reply:
no leh, I tried and it still failed tests

Reply:
oh wait I see what it's doing now lol okay will change it